packages/wrangler/src/config/validation.ts

```diff
@@ -2201,6 +2201,7 @@ const validateUnsafeBinding: ValidatorFn = (diagnostics, field, value) => {
 			"service",
 			"logfwdr",
 			"mtls_certificate",
+			"assets",
```

Comment:
is this `assets` or `experimental_assets`?

Reply:
That's the experimental assets binding - since EWC will be expecting the binding type to be 'assets', I've just used that immediately. Is a bit confusing though sorry!

Reply:
but the type of `experimental_assets.binding` is not exposed to end users. The shape we send to EWC is something entirely internal. Furthermore, this validation is part of `validateUnsafeBinding`, which is called as part of `validateUnsafeSettings`, which doesn't seem to be related to our work? AFAICT `validateUnsafeSettings` validates the non-inheritable `unsafe` config key, which does indeed expose the `type`, but is something entirely different
```
unsafe: {
	bindings?: {
		name: string;
		type: string;
		[key: string]: unknown;
	}[];
        ...
}
```

What this code here does, is basically to ensure that if users are trying to configure any non-inheritable bindings via `unsafe`, but that `wrangler.toml` supports directly, we error with the appropriate message. Since `experimental_assets` is inheritable, I don't think we should be adding it to this list

Reply:
whoops yeah I think I didn't dig through that enough and misunderstood - fixed